packages/client/src/client4.ts

```diff
@@ -4050,7 +4050,7 @@ export default class Client4 {
 
     // Client Helpers
 
-    private doFetch = async <ClientDataResponse>(url: string, options: Options): Promise<ClientDataResponse> => {
+    doFetch = async <ClientDataResponse>(url: string, options: Options): Promise<ClientDataResponse> => {
```

Comment:
Made public so I could use for `e2e/playwright/support/server/client.ts`.

Reply:
Done, please see https://github.com/mattermost/mattermost-webapp/pull/11468/commits/883848ec32a8e17b5a37f4c6c3a721efd49630d7